README.md

```diff
@@ -6,99 +6,157 @@
 [![Build Status](https://dev.azure.com/onnxruntime/onnxruntime/_apis/build/status/Linux%20GPU%20CI%20Pipeline?label=Linux+GPU)](https://dev.azure.com/onnxruntime/onnxruntime/_build/latest?definitionId=12)
 [![Build Status](https://dev.azure.com/onnxruntime/onnxruntime/_apis/build/status/MacOS%20CI%20Pipeline?label=MacOS+CPU)](https://dev.azure.com/onnxruntime/onnxruntime/_build/latest?definitionId=13)
 
-**ONNX Runtime** is an open-source scoring engine for Open Neural Network Exchange (ONNX) models.
+**ONNX Runtime** is an open-source high performance complete scoring engine for Open Neural Network Exchange (ONNX) models.
 
-ONNX is an open format for machine learning (ML) models that is supported by various ML and DNN frameworks and tools. This format makes it easier to interoperate between frameworks and to maximize the reach of your hardware optimization investments. Learn more about ONNX on [https://onnx.ai](https://onnx.ai) or view the [Github Repo](https://github.com/onnx/onnx).
+[ONNX](https://github.com/onnx/onnx) is an interoperable format for machine learning models supported by various ML and DNN frameworks and tools. The universal format makes it easier to interoperate between frameworks and maximize the reach of hardware optimization investments.
+***
+**Setup**
+* [Installation](#installation)
+* [APIs and Official Binaries](#apis-and-official-builds)
+* [Building from Source](#building-from-source)
 
-# Why use ONNX Runtime
-ONNX Runtime has an open architecture that is continually evolving to address the newest developments and challenges in AI and Deep Learning. ONNX Runtime stays up to date with the ONNX standard, supporting all ONNX releases with future compatibility and maintaining backwards compatibility with prior releases.
+**Getting Started**
+* [Deploying ONNX Runtime](#deploying-onnx-runtime)
+* [Getting ONNX Models](#getting-onnx-models)
+* [Examples and Tutorials](#examples-and-tutorials)
 
-ONNX Runtime continuously strives to provide top performance for a broad and growing number of usage scenarios in Machine Learning. Our investments focus on:
-1. Run any ONNX model
-2. High performance
-3. Cross platform
+**About ONNX Runtime**
```

Comment:
This should be above setup. Infact I prefer to add the content for Why use Onnxruntime here instead of a link...